libraries/aws_cloudtrail_trail.rb

```diff
@@ -10,11 +10,13 @@ class AwsCloudTrailTrail < AwsResourceBase
   EXAMPLE
 
   attr_reader :cloud_watch_logs_log_group_arn, :cloud_watch_logs_role_arn, :home_region, :trail_name,
-              :kms_key_id, :s3_bucket_name, :trail_arn, :is_multi_region_trail, :log_file_validation_enabled
+              :kms_key_id, :s3_bucket_name, :s3_key_prefix, :trail_arn, :is_multi_region_trail,
+              :log_file_validation_enabled, :is_organization_trail
 
   alias multi_region_trail? is_multi_region_trail
   alias log_file_validation_enabled? log_file_validation_enabled
   alias has_log_file_validation_enabled? log_file_validation_enabled
+  alias organization_trail? is_organization_trail
```

Comment:
Can we remove this? We dont require the alias. is_organization_trail looks good.

Reply:
I tried to run with the control with organization_trail, it failed. It is not taking the alias.

Code that I have used is 

control "test1" do
  describe aws_cloudtrail_trail('testbucketname') do
    its("is_organization_trail") { should eq false }
    its("organization_trail") { should eq false }
    its("organization_trail") { should eq "" }
  end
end

output

×  test1: CloudTrail testbucketname (2 failed)
     ✔  CloudTrail testbucketname is_organization_trail is expected to eq false
     ×  CloudTrail testbucketname organization_trail is expected to eq false

     expected: false
          got: #<#<Class:0x000000010fc0d860>::NullResponse:0x000000010fa83198>

     (compared using ==)
     ×  CloudTrail testbucketname organization_trail is expected to eq false

     expected: false
          got: #<#<Class:0x000000010fc0d860>::NullResponse:0x000000010fa83198>

     (compared using ==)

Reply:
@rezen Please check this.

Reply:
Try it { should be_organizational_trail } . It makes good English.

Reply:
it { should be_organizational_trail } 

This is working fine. So we can merge this PR.